api/src/main/java/org/openmrs/api/db/hibernate/HibernateConditionDAO.java

```diff
@@ -84,10 +72,22 @@ public Condition getConditionByUuid(String uuid) {
 	 */
 	@Override
 	public List<Condition> getConditionHistory(Patient patient) {
-		Query query = sessionFactory.getCurrentSession().createQuery(
-				"from Condition con where con.patient.patientId = :patientId and con.voided = false " +
-						"order by con.onsetDate desc");
-		query.setInteger("patientId", patient.getId());
+		Query<Condition> query = sessionFactory.getCurrentSession().createQuery(
+				"from Condition con where con.patient.patientId = :patientId " +
+						"order by con.dateCreated desc", Condition.class);
+		query.setParameter("patientId", patient.getId());
+		return query.list();
```

Comment:
So the idea here is that a condition history now *does* include voided conditions?

I guess the underlying question is, what is this condition history used for?  This has become more of an audit history of changes to the condition, right?